### PR TITLE
Tips for centos to resolve startup problem

### PR DESCRIPTION
**Pull request by netroby**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4755)

### CODE REVIEW

**Comment by netroby** [comment]
ping @zacharysarah 
Hello, this change will address centos 7 problem with kubeadm.

Will help people fall in trouble set up kubernetes with kubeadm.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -541,6 +541,39 @@ You may have trouble in the configuration if you see Pod statuses like `RunConta
    [`Vagrantfile`][ubuntu-vagrantfile] for how this can be achieved.
 
 [ubuntu-vagrantfile]: https://github.com/errordeveloper/k8s-playground/blob/22dd39dfc06111235620e6c4404a96ae146f26fd/Vagrantfile#L11)
+2. If you were using CentOS, while you get stuck while setup master node. you can follow this guide 
+
+```
```
Comment (zacharysarah) [inline docs/setup/independent/create-cluster-kubeadm.md]: Follow the style guide for [code blocks](https://kubernetes.io/docs/home/contribute/style-guide/#code-snippet-formatting). Make sure that code blocks contain code, not procedural information.

**Comment by netroby** [comment]
Hello, @zacharysarah Thanks for guide me
I made a new revision. 
May you please take a look?

**Comment by zacharysarah** [comment]
Approved with edits. 👍 

@netroby :wave: Thanks again for opening this PR. I encourage you to compare your [latest commit](https://github.com/kubernetes/kubernetes.github.io/pull/4755/commits/2eba685e6ba0458b3b056d5fa64e1d98ca540811) to the [final edits](https://github.com/kubernetes/kubernetes.github.io/pull/4755/commits/8300dc0d35c6d1c6d45ed1e14bab1e21532a461e) and [merge conflict resolution](https://github.com/kubernetes/kubernetes.github.io/pull/4755/commits/9907b9423a9fdf1fcf87844d1d51b876bd05c929). This may help improve your English and ease the approval of future PRs. 🙇

**Pull request merged**